docs/troubleshooting.md

```diff
@@ -0,0 +1,38 @@
+# Troubleshooting
+
+The `@jupyterlab/google-drive` extension is a complex plugin,
+made more so by the fact that it must make authenticated calls to Google's servers.
+This is a list of problems you may run into, with possible solutions.
+
+In order to better debug, it is always helpful to look at the Javascript console
+in your browser to see if any errors are being shown.
+
+### The Google Drive panel doesn't show a login button.
+
+This means that the plugin may not be loading the Google API libraries,
+or they may not be initializing correctly.
+Check your internet connection and look at the Javascript console
+to see if any errors are being shown.
+
+### I am seeing an error saying `Not a valid origin for the client`.
```

Comment:
Perhaps:  ~~I am seeing an error saying~~ `Not a valid origin for the client` error.